src/parse.rs

```diff
@@ -37,12 +35,10 @@ impl<const LEN: usize> FromHex for [u8; LEN] {
 
     fn from_hex(s: &str) -> Result<Self, Self::Error> {
         if s.len() == LEN * 2 {
-            let mut ret = ArrayVec::<u8, LEN>::new();
+            let mut ret = [0u8; LEN];
```

Comment:
This may have improved efficiency in one way but lost it in a different way. `ArrayVec` uses `MaybeUninit` inside, so should this.

Reply:
Promoted to #146